formats/financial_releases_geoblocker/frontend/examples/financial_releases_geoblocker.json

```diff
@@ -1,5 +1,5 @@
 {
-  "base_path": "/financial-releases-geoblocker",
```

Comment:
this is updated but github is not showing it in the conversation view of this PR because the file name has also changed to a more descriptive "financial_releases_campaign_geoblocker"